When I compile and test locally, I get the following error.
```
CMake Error at F:/gz-physics6/installed/x64-windows/share/cmake/gz-cmake3/cmake3/GzImportTarget.cmake:207 (set_target_properties):
  Property INTERFACE_LINK_LIBRARIES may not contain link-type keyword
  "debug".  The INTERFACE_LINK_LIBRARIES property may contain
  configuration-sensitive generator-expressions which may be used to specify
  per-configuration rules.
Call Stack (most recent call first):
  F:/gz-physics6/installed/x64-windows/share/cmake/gz-cmake3/cmake3/GzImportTarget.cmake:135 (_gz_sort_libraries)
  F:/gz-physics6/installed/x64-windows/share/cmake/gz-cmake3/cmake3/FindGzBullet.cmake:41 (gz_import_target)
  F:/gz-physics6/scripts/buildsystems/vcpkg.c
make:859 (_find_package)
  F:/gz-physics6/installed/x64-windows/share/cmake/gz-cmake3/cmake3/GzFindPackage.cmake:243 (find_package)
  CMakeLists.txt:84 (gz_find_package)

```
[config-x64-windows-dbg-err.log](https://github.com/microsoft/vcpkg/files/13110178/config-x64-windows-dbg-err.log)

--- a/ports/gz-physics6/dependencies.patch
+++ b/ports/gz-physics6/dependencies.patch
@@ -1,0 +1,21 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -73,9 +73,8 @@
+     collision-bullet
+     collision-ode
+     utils
+     utils-urdf
+   CONFIG
+   VERSION 6.9
+   REQUIRED_BY dartsim
+-  PKGCONFIG dart
+   PKGCONFIG_VER_COMPARISON >=)
+@@ -82,7 +82,6 @@
+ #--------------------------------------
+ # Find bullet for the bullet plugin wrapper
+ gz_find_package(GzBullet
+   VERSION 2.87
+   REQUIRED_BY bullet bullet-featherstone
+-  PKGCONFIG bullet
+   PKGCONFIG_VER_COMPARISON >=)

--- a/ports/gz-physics6/portfile.cmake
+++ b/ports/gz-physics6/portfile.cmake
@@ -1,0 +1,25 @@
+set(PACKAGE_NAME physics)
+
+ignition_modular_library(
+   NAME ${PACKAGE_NAME}
+   REF ${PORT}_${VERSION}
+   VERSION ${VERSION}
+   SHA512 c29594663509234e25c7d0a33848c0fe222c2b9471513978c18ea6873a17c66c43b4037c74e8849995fa6449c2dddc0f2ee669605893daf65119c277a17f39e1
+   OPTIONS 
+   PATCHES
+      dependencies.patch
+)
+
+if(VCPKG_TARGET_IS_WINDOWS)   
+   file(GLOB plugins "${CURRENT_PACKAGES_DIR}/lib/gz-physics-6/engine-plugins/*.dll")
+   if (NOT plugins STREQUAL "")
+      file(COPY ${plugins} DESTINATION "${CURRENT_PACKAGES_DIR}/bin/engine-plugins/")
+      file(REMOVE ${plugins})
+   endif()
+
+   file(GLOB plugins_debug "${CURRENT_PACKAGES_DIR}/debug/lib/gz-physics-6/engine-plugins/*.dll")
+   if (NOT plugins_debug STREQUAL "")
+      file(COPY ${plugins_debug} DESTINATION "${CURRENT_PACKAGES_DIR}/debug/bin/engine-plugins/")
+      file(REMOVE ${plugins_debug})
+   endif()
+endif()

--- a/ports/gz-physics6/vcpkg.json
+++ b/ports/gz-physics6/vcpkg.json
@@ -1,0 +1,22 @@
+{
+  "name": "gz-physics6",
+  "version": "6.5.1",
+  "description": "component of Gazebo, provides an abstract physics interface designed to support simulation and rapid development of robot applications.",
+  "homepage": "https://gazebosim.org/libs/physics",
+  "license": "Apache-2.0",
+  "dependencies": [
+    "bullet3",
+    "dartsim",
+    "eigen3",
+    "gz-cmake3",
+    "gz-common5",
+    "gz-math7",
+    "gz-plugin2",
+    "gz-utils2",
+    {
+      "name": "ignition-modularscripts",
+      "host": true
+    },
+    "sdformat13"
+  ]
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3180,6 +3180,10 @@
       "baseline": "9.5.0",
       "port-version": 0
     },
+    "gz-physics6": {
+      "baseline": "6.5.1",
+      "port-version": 0
+    },
     "gz-plugin2": {
       "baseline": "2.0.1",
       "port-version": 0

--- a/versions/g-/gz-physics6.json
+++ b/versions/g-/gz-physics6.json
@@ -1,0 +1,9 @@
+{
+  "versions": [
+    {
+      "git-tree": "9e84a89c5e37806e67295d51b6bd1ec565ccaede",
+      "version": "6.5.1",
+      "port-version": 0
+    }
+  ]
+}

